### DISCUSSION TITLE
Not able to set EC2 instance for Mumbai server

### DISCUSSION THREAD

**Discussion by saurabhhangotra** [getredash/redash#6555] [General]
I have used below AMI but its not running redash after creating instance, I also used AMI available in aws for redash 10 which I ran but i am getting login screen and not sure what email and password i need to use as its not showing setup screen
![image](https://github.com/getredash/redash/assets/76439768/3fa7ede9-e9ac-4a4d-94c4-7791adead16a)

**Comment by guidopetri** [comment]
The AMIs seem outdated (the screenshot you posted is for redash 8) so I'd recommend deploying via docker manually for now, instead.

**Reply by saurabhhangotra** [reply]
In documentation i can see redash 8 AMI, can you share any document or steps to deploy redash

**Reply by guidopetri** [reply]
I don't - you'd probably have to deploy similar to the docker compose [here](https://github.com/getredash/redash/blob/master/docker-compose.yml) but with lots of modifications.

**Reply by saurabhhangotra** [reply]
I tried installing through docker but still facing issue when run and try to open redash
![image](https://github.com/getredash/redash/assets/76439768/2b10d74f-950e-41b4-9154-5e10431e05e3)

**Reply by justinclift** [reply]
That error message looks like you forgot to initialise the local PostgreSQL database before starting things up?

```
$ docker compose run server create_db
```

Then start Redash with:

```
$ docker compose up -d
```

If you're using an older release of docker, then you'll probably need to use `docker-compose` instead of `docker compose`.  (The `docker compose` command itself changed recently.)
